index.html

```diff
@@ -5931,7 +5931,8 @@ <h5>Note regarding the ARIA 1.1 <code>none</code> role.</h5>
 		<div class="role" id="note">
 			<rdef>note</rdef>
 			<div class="role-description">
-				<p>A section whose content is parenthetic or ancillary to the main content of the resource.</p>
+				<p>A <rref>section</rref> whose content qualifies, explains, annotates or is subordinate to some other piece of content</p>
```

Comment:
>qualifies, explains, annotates or is subordinate

do not think "annotates" or "subordinate" should be added here.  i understand why they would be... but their inclusion could muddle `note` with `comment` or `suggestion`, if not also `doc-footnote` or `doc-endnote`.

suggest "supplementary" is more appropriate, if we even need additional words beyond "qualifies" and "explains"

Reply:
Good. I replaced that mini-thesaurus wiith "supplementary".